contributing.md

```diff
@@ -0,0 +1,3 @@
+# Contributing Guide
+
+See our the Brigade project [Contributing Guide](https://github.com/brigadecore/community/blob/master/contributing.md).
```

Comment:
Suggest removing "the" so it reads "See our Brigade project Contributing Guide".

Reply:
updated